src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -508,7 +516,15 @@ static inline struct mlx5_grh_av *uct_dc_mlx5_ep_get_grh(uct_dc_mlx5_ep_t *ep)
 
 #define UCT_DC_MLX5_CHECK_RES(_iface, _ep) \
     { \
-        ucs_status_t status = uct_dc_mlx5_iface_dci_get(_iface, _ep); \
+        ucs_status_t status = uct_dc_mlx5_iface_dci_get(_iface, _ep, 1); \
+        if (ucs_unlikely(status != UCS_OK)) { \
+            return status; \
+        } \
+    }
+
+#define UCT_DC_MLX5_CHECK_DCI_RES(_iface, _ep) \
```

Comment:
what this macro is named as "CHEC_DCI_RES" if we don't check iface res?

Reply:
dci is dci initiator (QP), we check just QP resources